code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -531,6 +531,33 @@
 	tastes = list("paint thinner" = 1)
 	color = "#EE35FF"
 
+/obj/item/reagent_containers/food/snacks/chewable/bubblegum/bubblegum
+	name = "bubblegum gum"
+	desc = "A rubbery strip of gum. You don't feel like eating it is a good idea."
+	color = "#913D3D"
+	list_reagents = list(/datum/reagent/blood = 15)
```

Comment:
Maybe some reagent that indirectly replenishes blood instead, not unlike iron?

Reply:
that would be less thematic